### PR TITLE
Update info log of bundle activation to debug log and add exception handling

### PR DESCRIPTION
**Pull request by Osara-B**
## Purpose
Update info log of bundle activation to debug log
Add exception handling

* https://github.com/wso2/product-is/issues/20778

### CODE REVIEW

File: components/org.wso2.carbon.extension.identity.verification.mgt/src/main/java/org/wso2/carbon/extension/identity/verification/mgt/internal/IdentityVerificationServiceComponent.java [hunk 1]
```diff
@@ -67,7 +67,6 @@ protected void activate(ComponentContext ctxt) {
             ctxt.getBundleContext().registerService(UserOperationEventListener.class.getName(),
                     idVUserOperationEventListener, null);
 
-            log.info("IdentityVerificationService bundle activated successfully.");
             if (log.isDebugEnabled()) {
                 log.debug("IdentityVerificationService bundle is activated");
```
Comment (darshanasbg) [inline components/org.wso2.carbon.extension.identity.verification.mgt/src/main/java/org/wso2/carbon/extension/identity/verification/mgt/internal/IdentityVerificationServiceComponent.java]: Lets use consistent message format.

File: components/org.wso2.carbon.extension.identity.verification.provider/src/main/java/org/wso2/carbon/extension/identity/verification/provider/internal/IdVProviderMgtServiceComponent.java [hunk 2]
```diff
@@ -49,12 +49,19 @@ public class IdVProviderMgtServiceComponent {
...
+            ctxt.getBundleContext().registerService(IdVProviderDAO.class.getName(),
+                    new CachedBackedIdVProviderDAO(idVProviderDAO), null);
+            ctxt.getBundleContext().registerService(IdVProviderManager.class.getName(),
+                    new IdVProviderManagerImpl(), null);
+
+            if (log.isDebugEnabled()) {
+                log.debug("IdVProviderMgtService bundle activated successfully.");
+            }
+        } catch (Throwable e) {
+            log.fatal("Error while activating IdVProviderMgtService bundle");
```
Comment (darshanasbg) [inline components/org.wso2.carbon.extension.identity.verification.provider/src/main/java/org/wso2/carbon/extension/identity/verification/provider/internal/IdVProviderMgtServiceComponent.java]: `log.fatal("Error while activating IdVProviderMgtService bundle", e);`

Comment (Osara-B) [reply components/org.wso2.carbon.extension.identity.verification.provider/src/main/java/org/wso2/carbon/extension/identity/verification/provider/internal/IdVProviderMgtServiceComponent.java]: Resolved in [Update info log of bundle activation to debug log and add exception h…](https://github.com/wso2-extensions/identity-verification/pull/17/commits/485d0b5e43a25b1edae4b04ed8e8896b4640c1b0)

File: components/org.wso2.carbon.extension.identity.verification.mgt/src/main/java/org/wso2/carbon/extension/identity/verification/mgt/internal/IdentityVerificationServiceComponent.java (hunk 1 above)
Comment (Osara-B) [reply components/org.wso2.carbon.extension.identity.verification.mgt/src/main/java/org/wso2/carbon/extension/identity/verification/mgt/internal/IdentityVerificationServiceComponent.java]: Resolved in [Update info log of bundle activation to debug log and add exception h…](https://github.com/wso2-extensions/identity-verification/pull/17/commits/485d0b5e43a25b1edae4b04ed8e8896b4640c1b0)

**Review by darshanasbg — APPROVED** [review]

**Pull request merged**